packages/react/src/checkout/hooks/useCheckout.ts

```diff
@@ -476,6 +481,46 @@ function useCheckout(
     reset();
   }
 
+  const {
+    data: { countryAddressSchemas },
+    actions: { fetch: fetchCountryAddressSchemas },
+  } = useCountryAddressSchemas(
+    (checkoutOrder?.shippingAddress?.country.alpha2Code as string) || '',
+    { enableAutoFetch: false },
+  );
+
+  const isShippingAddressZipCodeValid = useCallback(
+    async (shippingAddress?: CheckoutAddress) => {
+      let addressSchema;
+
+      addressSchema = countryAddressSchemas;
+
+      const tempShippingAddress =
+        shippingAddress || checkoutOrder?.shippingAddress;
+
+      if (!tempShippingAddress) {
+        return { isValid: false };
+      }
+
+      if (!addressSchema) {
+        addressSchema = await fetchCountryAddressSchemas(
+          tempShippingAddress?.country.alpha2Code,
+        );
+        console.log(
```

Comment:
Search for other console.log/traces you might have and remove them, please.

Reply:
sorry, already removed